deployment/src/main/java/io/quarkiverse/jasperreports/deployment/JasperReportsProcessor.java

```diff
@@ -14,12 +19,42 @@ class JasperReportsProcessor {
 
     private static final String FEATURE = "jasperreports";
     private static final String EXTENSIONS_FILE = "jasperreports_extension.properties";
+    private static final String DEFAULT_ROOT_PATH = "";
 
     @BuildStep
     FeatureBuildItem feature() {
         return new FeatureBuildItem(FEATURE);
     }
 
+    @BuildStep
+    ReportRootBuildItem defaultReportRoot() {
+        return new ReportRootBuildItem(DEFAULT_ROOT_PATH);
+    }
+
+    @BuildStep
+    ReportRootsBuildItem collectReportRoots(List<ReportRootBuildItem> reportRoots) {
+        Set<String> roots = new HashSet<>();
```

Comment:
Maybe use LinkedHashSet to keep them in order?

Reply:
Or maybe consume the List directly?

Reply:
Consuming the list directly, though we're only producing one item in that list... :thinking: